Cython/Compiler/StringEncoding.py

```diff
@@ -146,6 +146,16 @@ def as_c_string_literal(self):
             s = bytes_literal(self.byteencode(), self.encoding)
         return s.as_c_string_literal()
 
+    if sys.version_info[0] == 2:
+        def isascii(self):
```

Comment:
Nice Idea. I didn't even think of making it a method, but sure, makes total sense.